mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt

```diff
@@ -426,5 +438,29 @@ class WidgetImageView(context: Context, attrs: AttributeSet?) : AppCompatImageVi
 
     companion object {
         private val TAG = WidgetImageView::class.java.simpleName
+        private fun createConfig(context: Context) = SkeletonConfig.default(context).apply {
+            maskColor = context.resolveThemedColor(R.attr.skeletonBackground, maskColor)
+            shimmerColor = context.resolveThemedColor(R.attr.skeletonShimmer, shimmerColor)
+        }
     }
 }
+
+internal interface WidgetImageViewIntf {
```

Comment:
What does the `Intf` mean?

Reply:
Interface. I can also spell it out if you prefer.

Reply:
Ok, I'm fine with that.